src/react/components/common/assetPreview/videoAsset.tsx

```diff
@@ -138,38 +153,97 @@ export class VideoAsset extends React.Component<IVideoAssetProps> {
     }
 
     /**
-     * Seeks the video to the timestamp specified in the component properties
+     * @name - Move to the next expected frame
+     * @description - Moves the videos current position forward one frame based on the current
+     * project settings for frame rate extraction
      */
-    private seekToTimestamp = () => {
-        if (this.props.timestamp > 0) {
-            this.videoPlayer.current.pause();
-            this.videoPlayer.current.seek(this.props.timestamp);
+    private moveNextExpectedFrame = () => {
```

Comment:
I mentioned a similar comments in @PIC123's PR.  Should the sudo key frames be calculated from the absolute beginning of the video vs the current player time?  I don't know what the answer is but i know the frames would start getting offset if you manually skipped around in the video and then also utilized this skip to previous/next frame feature.

Reply:
I can see the argument both ways. I don't feel super strongly about either honestly, we should just make a call and do that.

Reply:
Updated to lock to video key frame rate per discussion